src/main.js

```diff
@@ -9,23 +9,55 @@ var coverTitle = document.querySelector(".cover-title");
 var tagLine1 = document.querySelector(".tagline-1");
 var tagLine2 = document.querySelector(".tagline-2");
 var randomCover = document.querySelector(".random-cover-button")
-var makeYourOwn = document.querySelector(".make-new-button")
+var makeYourOwnButton = document.querySelector(".make-new-button")
```

Comment:
Good job on making the names more explicit, reduces any potential confusion. 👍

Reply:
Nice descriptive variable names. Maybe just be a little more consistent with your semi-colons in your entire main.js file. Some lines end with semi-colons and some do not and its best to just choose to either include them entirely or not